user-service/src/main/java/rs/edu/raf/banka1/model/MarketOrder.java

```diff
@@ -0,0 +1,29 @@
+package rs.edu.raf.banka1.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class MarketOrder {
```

Comment:
Je l smo beše pričali da ovo preimenujemo u Order?

Reply:
ne moze, kljucna rec

Reply:
BaseOrder?